tas-initializer/Dockerfile.tas-initializer

```diff
@@ -0,0 +1,25 @@
+FROM registry.redhat.io/rhel9/go-toolset@sha256:f969e6a4fe53663f3fc8d0ddc17ec2fc3def2af8e68ce7061f016a598798f474 AS build-env
```

Comment:
This should probably be ubi-minimal instead of go-toolset, unless you are planning further work.

Reply:
ubi-minimal lacks tar for the current installation method of OC (will discuss that as a separate point), so it can be changed to ubi9/ubi but not ubi9/ubi-minimal